phabfive/paste.py

```diff
@@ -5,15 +5,36 @@
 
 # phabfive imports
 from phabfive.core import Phabfive
+from phabfive.exceptions import PhabfiveDataException
 
 class Paste(Phabfive):
     def __init__(self):
         super(Paste, self).__init__()
 
-    def get_pastes(self):
+    def get_pastes(self, queryKey="all", attachments=None, constraints=None):
```

Comment:
A secondary comment @maiyuki is that using a string as the default value for a kwarg variable into a method is a big nono to do. Sure it might look more complicated to have variable validation on the inside for types and values. But python strongly discourage from using this way. Basically copy what you have done with attachments and constraints variable.

Reply:
This is my fault. Again, I will make a new PR to fix this. This PR is good to go.